src/liboslexec/wide/wide_opcolor.cpp

```diff
@@ -309,13 +309,13 @@ wide_transformc (ColorSystem cs, StringParam fromspace, StringParam tospace,
     Wide<COLOR> wCrgb(bCrgb);
     if (fromspace == STRING_PARAMS(RGB) || fromspace == STRING_PARAMS(rgb)
          || fromspace == STRING_PARAMS(linear) || fromspace == cs.colorspace()) {
-        OSL_OMP_PRAGMA(omp simd simdlen(__OSL_WIDTH))
+        OSL_OMP_COMPLEX_SIMD_LOOP(simdlen(__OSL_WIDTH))
```

Comment:
To not loose vectorization for clang 13 and below, you might want to conditionally use 
```
#if OSL_CLANG_VERSION > 140000
OSL_OMP_COMPLEX_SIMD_LOOP(simdlen(__OSL_WIDTH))
#else
OSL_OMP_PRAGMA(omp simd simdlen(__OSL_WIDTH))
#endif
```
This does appear to be a regression in Clang14 of some kind.  Its odd that these color operations, especially this super simple assignment loop, are having issues (where as much more complex noises and spline operations continue to vectorize).  So I suspect it has something to do with the underlying Color3 class itself, of the organization/optimization of wide_transformc.  Given that other functions in wide_opcolor.cpp are vectorizing fine using Color3, that leads me to think its wide_transformc itself.  
I don't have an easy way to reproduce without moving to clang14 myself.

Some experiments to try:
1.  Change ColorSystem parameters to be const & vs. copy by value (this looks like it might be an existing issue):
```
wide_transformc (ColorSystem cs, ...
```
to 
```
wide_transformc (const ColorSystem &cs, ...
```
1.  SIMD loop isolation.  I have seen compilers try to do loop fusion and other magic that messes up otherwise valid vectorizing loops.  We do this in wide_opmatrix.cpp utilizing a simple noinline helper functor called invoke.  This "isolation" technique can also help with register allocation by limiting the scope of what the optimizer has to deal with in a single function:
```
namespace {

// invoke is helper to ensure a functor is compiled inside a
// actual function call vs. inlining.
// Useful for keeping the slow path from fusing
// with a streamlined SIMD loop or influencing
// register usage
template<typename FunctorT>
OSL_NOINLINE void
invoke(FunctorT f);

template<typename FunctorT>
void
invoke(FunctorT f)
{
    f();
}
}
```
then inside of wide_transformc wrap each simd loop in its own invoke call with a lambda expression
```
        invoke([=]() -> void {
            OSL_OMP_PRAGMA(omp simd simdlen(__OSL_WIDTH))
            for (int lane = 0; lane < __OSL_WIDTH; ++lane) {
                COLOR C = wInput[lane];
                wCrgb[lane] = C;
            }
        });
```
for the loops that use the ColorSystem argument `cs` might want to add that as a reference in the lamba capture clause.
```
        invoke([=,&cs]() -> void {
```